### PR TITLE
[ES] Translation for Error Handling Lesson

### PR DESCRIPTION
**Pull request by aleandros**

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @aleandros!  @setzer777 @erickgnavar could either of you take a look at this translation?

File: es/lessons/advanced/error-handling.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+iex> try do
+...>   raise "Oh no!"
+...> rescue
+...>   e in RuntimeError -> IO.puts("An error occurred: " <> e.message)
+...> end
+An error occurred: Oh no!
+:ok
+```
+
+Es posible coincidir múltiples errores en un sólo `rescue`:
```
Comment (erickgnavar) [inline es/lessons/advanced/error-handling.md]: No es necesaria la tilde en `solo`, ya que no hay ambigüedad

**Comment by erickgnavar** [comment]
Look good, I added one annotation
Thanks @aleandros

**Comment by aleandros** [comment]
Done, @erickgnavar! Thanks

**Comment by setzer777** [comment]
This looks awesome for me @aleandros

**Comment by doomspork** [comment]
Thank you @aleandros, @erickgnavar, and @setzer777 👍

**Pull request merged**